runtime/src/main/java/org/corfudb/runtime/view/metadata/IMetadataServiceView.java

```diff
@@ -0,0 +1,40 @@
+package org.corfudb.runtime.view.metadata;
+
+import java.util.UUID;
+
+/**
+ * This interface defines the operations provided by the metedata service.
+ */
+
+public interface IMetadataServiceView {
+
+    /**
+     * Create a stream
+     * @param stream The stream to create
+     * @throws StreamAlreadyExistsException This exception is thrown if a previous create opertaion
+     * succeeded with the same stream uuid
+     */
+    void createStream(UUID stream) throws StreamAlreadyExistsException;
```

Comment:
One way would be to acquire an address first - I think you have to provide a start address, otherwise what would the flow for creating a stream be?